r_exercise5.R

```diff
@@ -17,10 +17,13 @@ ggplot(data = ess,
         ylab("Number of observations") +
         theme_bw()
 
+# Missed a spot! What happens if you try out different bin numbers?
```

Comment:
On exercise 5 I had written about the bins "# Changing bins lower; less bars, higher; more bars", maybe it wasn't clear.
But I know what happens when you increase/decrease the number of bins anyhow ;)